src/main/haskell/kore/src/Kore/Step/Simplification/Application.hs

```diff
@@ -207,13 +205,13 @@ makeExpandedApplication tools symbol children
         , _proof) <-
             mergePredicatesAndSubstitutions
                 tools
-                (map ExpandedPattern.predicate children)
-                (map ExpandedPattern.substitution children)
+                (map predicate children)
```

Comment:
You shouldn't use Predicate's getters unqualified (see the coding standards, naming, records).

The previous kind of imports, which split the ExpandedPattern in two, an unqualified import for the type and the constructor, and a qualified import for everything in the ExpandedPattern type, made sure that this unqualified access can't happen by mistake.

Of course, you don't have to import it like that, but you need to qualify these getters somehow.

Please fix this and the other getters in this pull request.

Reply:
That's actually a really weird issue. The fact that the name `type ExpandedPattern` coincides with the module name makes it confused.

Reply:
True. That's why I recommend (but do not enforce since the style guide does not require it) the splitted imports I mentioned above.

Reply:
I can still prefix it with `ExpandedPattern` instead of `Predicated`, which is misleading but w/e. I suppose in a future PR the module will be renamed `Predicated`.